static/assets/js/core.js

```diff
@@ -106,15 +131,15 @@ $(document).ready(function() {
   }
 
   /** Dark mode controller */
-  if (store.get('dark-mode') === true) {
```

Comment:
This should be in a different file indeed, I must have missed it in the previous PR (the one that introduced the dark mode)